engine/src/main/java/org/terasology/rendering/dag/nodes/OutlineNode.java

```diff
@@ -68,28 +62,16 @@
     private float sceneOpaqueFboWidth;
     private float sceneOpaqueFboHeight;
 
-    @Range(min = 0.0f, max = 16.0f)
-    float pixelOffsetX = 1.0f;
-    @Range(min = 0.0f, max = 16.0f)
-    float pixelOffsetY = 1.0f;
-
-    private SubmersibleCamera activeCamera;
```

Comment:
This Sample ShaderParameters clean-up commit gives me a good idea of what you intend to do and it's in line with my comments so far. Please go ahead!